src/components/organisms/HelpCenter/HelpCenter.tsx

```diff
@@ -25,18 +27,31 @@ export const HelpCenter: React.FC = () => {
     [startLoading]
   );
 
+  const handleBackClicked = () => {
```

Comment:
`useCallback` ?

Reply:
fixed in ab0afbed983975054e3dea0f78619a35e5629d8c